mock/mockimpl/kvproc/basic.go

```diff
@@ -57,9 +57,13 @@ func (e *Engine) parseExpiry(expiry uint32) time.Time {
 
 	// TODO(brett19): Check if this is the right edge for expiry.
 	if expiry > 30*24*60*60 {
-		return time.Unix(int64(expiry), 0)
+		return time.Unix(int64(expiry), 0).Add(e.db.Chrono().TimeShift())
```

Comment:
This is a little... strange, but I guess it's the best version of strangeness...